src_blog/_posts/Grain-State-of-the-Union-2020.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Grain State of the Union 2020
+subtitle: Every update you need to know, and what's to come in 2021.
+date: 2020-09-29 12:00:00
+author: Oscar Spencer
+cover: /blog/photos/cover/GrainStateOfTheUnionCover.jpg
+coverAttribution: Luke Chesser / Unsplash
```

Comment:
Did you edit a photo by him with the logo on it?

Reply:
I suppose I could make my own gradient, but I was lazy.